.github/workflows/release.yml

```diff
@@ -17,11 +17,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        build: [linux-musl, macos-x86_64, macos-aarch64, win-msvc]
+        build: [linux-x86_64-musl, linux-aarch64-musl, macos-x86_64, macos-aarch64, win-msvc]
         include:
-        - build: linux-musl
+        - build: linux-x86_64-musl
           os: ubuntu-20.04
           target: x86_64-unknown-linux-musl
+        - build: linux-aarch64-musl
+          os: ubuntu-24.04
```

Comment:
Since we're producing static musl binaries anyway, they aren't dependent on the version of glibc users will have; so let's just go ahead and move the x86-64 build to 24.04, too.